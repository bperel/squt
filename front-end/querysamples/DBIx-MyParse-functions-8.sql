SELECT '1997-12-31 23:59:59' + INTERVAL 1 SECOND
