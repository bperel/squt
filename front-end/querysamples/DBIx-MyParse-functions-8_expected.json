{
  "warning": {
    "no alias": {
      "date_add_interval": "select"
    }
  },
  "functions": {
    "0": {
      "to": "output",
      "constants": {
        "1": "1",
        "interval_second": "interval_second",
        "1997-12-31 23:59:59": "1997-12-31 23:59:59"
      },
      "name": "date_add_interval",
      "alias": "0"
    }
  }
}